packages/nx-oc/src/generators/deployment/deployment.ts

```diff
@@ -28,13 +28,19 @@ function normalizeOptions(
   
   // TODO: Find a better way to determine this.
   const config = readProjectConfiguration(host, projectName);
-  const appType = config.targets.build.executor === '@nrwl/web:build' ?
-    'frontend' : 
-    (
-      config.targets.build.executor === '@nrwl/node:build' ? 
-        'express' : 
-        'dotnet'
-    );
+  let appType = null;
+  switch (config.targets.build.executor) {
+    case '@nrwl/web:build':
```

Comment:
do you need a break; here? or some sort of logic

Reply:
For now those two cases (react and angular) are treated the same. In the future we may want to differentiate between them.

Reply:
ok